frontend/src/app/exploration/exploration.module.ts

```diff
@@ -0,0 +1,18 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { FormsModule } from '@angular/forms';
+import { MaterialModule } from '../material.module';
+import { TranslateModule } from '@ngx-translate/core';
+
+
+@NgModule({
+  declarations: [    
+  ],
+  imports: [
+    CommonModule,
+    FormsModule,
+    MaterialModule,
+    TranslateModule.forChild()
+  ]
+})
+export class ExplorationModule { }
```

Comment:
Is this module needed for anything in the future?

Reply:
So initially, we split the map into upload-map and search-map; with the search-map being part of the `ExplorationModule`. If you want to use this module for your stuff, I can let it in, otherwise I can also delete it.